inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java

```diff
@@ -49,7 +49,7 @@
 public class PulsarOperator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InlongClusterServiceImpl.class);
-    private static final int MAX_PARTITION = 100;
+    private static final int MAX_PARTITION = 1000;
```

Comment:
Please add some comment for the value, such as:
```suggestion
    /**
     *The maximum number of partitions, which is an empirical value,
     * generally does not exceed 1000 in large clusters.
     */
    private static final int MAX_PARTITION = 1000;
```